data_format_config.go

```diff
@@ -39,6 +40,30 @@ func NewCSVOptions(opts map[string]interface{}) (FormatConfig, error) {
 		}
 	}
 
+	if opts["lazyQuotes"] != nil {
+		if lq, ok := opts["lazyQuotes"].(bool); ok {
+			o.HeaderRow = lq
+		} else {
+			return nil, fmt.Errorf("invalid lazyQuotes value: %s", opts["lazyQuotes"])
+		}
+	}
+
+	if opts["separator"] != nil {
+		if sep, ok := opts["separator"].(string); ok {
+			o.Separator = rune(sep[0])
```

Comment:
add a check to ensure that separator has length 1

Reply:
ugh I *knew* I shoulda added that in 😄